src/otx/core/data/transform_libs/torchvision.py

```diff
@@ -125,6 +128,159 @@ def _get_preprocess_shape(self, oldh: int, oldw: int, long_side_length: int) ->
         return (newh, neww)
 
 
+class DecodeVideo(tvt_v2.Transform):
```

Comment:
I'd rather move the decode process to `Dataset._get_item() -> tv_tensor.Video`, not in the transform pipeline. Can you do this work at the next PR?

Reply:
Sure I'll take a look at this on next PR